pyfr/backends/openmp/kernels/par-memcpy.mako

```diff
@@ -2,16 +2,14 @@
 <%inherit file='base'/>
 <%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
 
-#include <string.h>
```

Comment:
This is needed for memcpy?

Reply:
I don't think so as I don't use char pointer any more.

Reply:
But the memcpy function is defined in string.h

Reply:
It doesn't complain and it works. Maybe its the compiler. I'll put it back in any case then.